test/studies/adventOfCode/2022/day11/bradc/day11-par.chpl

```diff
@@ -0,0 +1,181 @@
+use IO, List, Barriers;
+
+config const numRounds = 20;
+
+// things that define a monkey
+class Monkey {
+  var id: int,                 // an integer ID
+      Items: [0..1] list(int, parSafe=true),  // two lists of its items
+      current = 0, next=1,     // which list is the current vs. next one
+      op: owned MathOp,            // the operator the monkey does
+      divisor: int,            // the divisor for its % operation check
+      targetMonkey: 2*int,     // the target monkeys it throws items to
+      numInspected: int;       // the count of how many inspections we've done
+}
+
+/*
```

Comment:
I assume you plan to remove this block of comments.

Reply:
Yeah, and all the rest of the comments (since the article will do that).